### PR TITLE
RSWEB-8845: modularized events as global component

### PR DESCRIPTION
**Pull request by runrog**
New page here, now with header/footer
http://localhost:9000/derek/view-templates/events/events

### CODE REVIEW

**Comment by runrog** [comment]
Resolving conflicts and rebasing

File: styleguide/derek/view-templates/events/events.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+jQuery(document).ready(() => {
+  jQuery('.rsFilter-bar').rsFilterBar();
+});
```
Comment (WidgetsBurritos) [inline styleguide/derek/view-templates/events/events.js]: For consistency, do you wanna swap this internal jQuery reference out like this? 
```
jQuery(document).ready(($) => {
  $('.rsFilter-bar').rsFilterBar();
});
```

File: styleguide/global/global.js [hunk 2]
```diff
@@ -78,7 +195,10 @@ jQuery(document).ready(($) => {
 
   // Zoolander only solution for auto collapsing menu on example pages
   const url = window.location.pathname;
-  if ((url.match(/\/derek\/incubation\//gi) && url !== '/derek/incubation/') || (url.match(/\/derek\/solutions\//gi) && url !== '/derek/solutions/') || (url.match(/\/derek\/templates\//gi) && url !== '/derek/templates/')) {
+  if ((url.match(/\/derek\/incubation\//gi) && url !== '/derek/incubation/') ||
+      (url.match(/\/derek\/view-templates\//gi) && url !== '/derek/incubation/') ||
+      (url.match(/\/derek\/solutions\//gi) && url !== '/derek/solutions/') ||
+      (url.match(/\/derek\/templates\//gi) && url !== '/derek/templates/')) {
```
Comment (WidgetsBurritos) [inline styleguide/global/global.js]: thank you.  this is much more readable.

File: styleguide/derek/view-templates/events/events.js (hunk 1 above)
Comment (runrog) [inline styleguide/derek/view-templates/events/events.js]: we could. I'll test it out

File: styleguide/global/global.js (hunk 2 above)
Comment (WidgetsBurritos) [inline styleguide/global/global.js]: That said, this doesn't appear to be working for me. It's staying open when I load it.
image

Also that second part of the URL doesn't make sense here: 
```
(url.match(/\/derek\/view-templates\//gi) && url !== '/derek/incubation/') 
```

File: styleguide/derek/view-templates/events/events.js (hunk 1 above)
Comment (WidgetsBurritos) [inline styleguide/derek/view-templates/events/events.js]: k thanks

File: styleguide/global/global.js (hunk 2 above)
Comment (WidgetsBurritos) [inline styleguide/global/global.js]: oh nevermind, i realize why it's not working.  i'm gonna ping you separately.

Comment (runrog) [inline styleguide/global/global.js]: oh you're right. I will fix that

**Pull request merged**